scripts/bump_cairo_bindings.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -n "$(git status --porcelain)" ]; then
+  >&2 echo "Working tree has uncommited changes, please stash/commit them before bumping submodule version"
+  exit 1
+fi
+
+tracking_branch="$(git for-each-ref --format='%(upstream:short)' "$(git symbolic-ref -q HEAD)")"
+
+if [ $tracking_branch == "origin/master" ] ; then
+  >&2 echo "You are currently tracking master branch. Checkout to another branch before bumping submodule version"
```

Comment:
shouldn't we do something like `git fetch --all` first? Or did I miss something?

Reply:
For what reason?

Reply:
Let's say you install the bindings, then the master of the bindings is updated and then you run this script. This check is going to pass, right? Maybe I missed the concept, so in such a case, we're supposed to run install script again?

Reply:
> in such a case, we're supposed to run the install script again?

yes. it's mentioned in readme

Reply:
Ok, thx